bin/setup

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
+# ========== Install mysql ==========
+echo "Checking MySQL"
+
+# Check for existing mysql
+mysql_exists=`brew list -1 | grep mysql`
```

Comment:
Since this file is Mac-only, can we rename this file to setup_mac? And die with an appropriate message if `uname` ≠ Darwin?